File: ff-frederiksen/Sarcasm.java
import java.lang.StringBuilder;

public class Sarcasm {


public static void main(String[]args){
    String text = "this does not align with our core values";
        
    StringBuilder builder = new StringBuilder();


    for (int i = 0; i < text.length(); i++){
            if ( i % 2 == 0){
                builder.append(returnUppercase(text.charAt(i)));
            }
            else{
                builder.append(returnLowercase(text.charAt(i)));
            }

    }
    String result = builder.toString();
    System.out.println(result);
}

public static char returnUppercase(char c){

    return Character.toUpperCase(c);
}

public static char returnLowercase(char c){
    return Character.toLowerCase(c);
}
}

File: nish-d/Sarcasm.kt
fun main() {
    val text = "this does not align with our core values"

        val builder = StringBuilder()
		var isUpper = true
		builder.append(text[0])
        for (i in 1 until text.length) {
            if(text[i-1]>='a' && text[i-1]<='z'){
                isUpper = !isUpper
            }
            if(text[i]>='a' && text[i]<='z'){
            	if (isUpper) {
                	builder.append(text[i].toUpperCase())
            	} else {
                	builder.append(text[i].toLowerCase())
            	}
            }else{
                builder.append(text[i])
            }
        }
        val result = builder.toString()
        println(result)
}